src/libraries/Microsoft.PowerFx.Core/Public/Values/BlankValue.cs

```diff
@@ -32,6 +32,12 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
+            if (settings.Compact)
+            {
+                sb.Append(new BlankType().DefaultExpressionValue());
```

Comment:
>sb.Append(new BlankType().DefaultExpressionValue()); [](http://example.com/codeflow?start=16&length=52)

Merge into the `if` clause below (`if (Type is BlankType || settings.Compact)`

Reply:
Just to be explicit, this should be: 
sb.Append("Blank()")

Because it's possible that BlankType.DefaultExpression could change and get more complex.

Reply:
> Merge into the if clause below - if (Type is BlankType || settings.Compact)

These can't be merged due to the fact that we have a Blank value but not necessarily a Blank type. In the event we have "**Compact == true**" and "**Type is DateType**", the outcome would be "**DateValue("0001-01-01T00:00:00.0000000Z")**" which is the default value for a date type. 
If "**Compact == true**", we can ignore the Type and simply return the blank value representation ("**Blank()**").